zcashd-tor.sh

```diff
@@ -0,0 +1,21 @@
+function sig_handle {
+  killall zcashd
+  exit 1
+}
+trap sig_handle SIGTERM
```

Comment:
this just catches a parent process sending kill / term, not a failure in the script, right?

Reply:
Correct